projects/vdk-plugins/vdk-impala/tests/conftest.py

```diff
@@ -53,3 +53,4 @@ def impala_service(docker_ip, docker_services):
     docker_services.wait_until_responsive(
         timeout=90.0, pause=0.3, check=lambda: _is_responsive(runner)
     )
+    time.sleep(10)
```

Comment:
Shouldn't those wait workarounds be encapsulated within _is_responsive(), to guarantee responsiveness?

Reply:
The issue is that _is_responsive doesn't guarantee it, since a 0 exit code is returned before the service is up.